packages/mdc-dialog/package-lock.json

```diff
@@ -1,8 +1,6 @@
 {
-  "name": "@material/dialog",
-  "version": "1.1.0",
-  "lockfileVersion": 1,
   "requires": true,
```

Comment:
Please revert all package-lock.json changes. Since you're not modiying dependencies of any packages these file changes are not relevant.